src/query/query_dump.c

```diff
@@ -3274,6 +3275,21 @@ qdump_print_stats_text (FILE * fp, xasl_node * xasl_p, int indent)
   qdump_print_stats_text (fp, xasl_p->scan_ptr, indent);
   qdump_print_stats_text (fp, xasl_p->connect_by_ptr, indent);
 
+  if (xasl_p->sq_cache && XASL_IS_FLAGED (xasl_p, XASL_SQ_CACHE))
+    {
+      fprintf (fp, "%*c", indent, ' ');
+      if (xasl_p->sq_cache->enabled)
+	{
+	  fprintf (fp, "SUBQUERY_CACHE (hit: %d, miss: %d, size: %lu Bytes, status: enabled)\n",
```

Comment:
The words "hit," "miss," and "status" are printed in lowercase letters. However, the word "Bytes" is not printed in lowercase.
How about using "B" instead of the word "Bytes" or printing it in lowercase?

Reply:
Okay.